packages/flutter/test/material/outline_button_test.dart

```diff
@@ -236,9 +236,9 @@ void main() {
         child: Material(
           child: Center(
             child: OutlineButton(
-                key: buttonKey,
-                onPressed: () { },
-                child: const Text('ABC'),
+              key: buttonKey,
+              onPressed: () { },
```

Comment:
Remove space between the {}

Reply:
Done, and elsewhere in this file.  I'm not sure, but I think Hans prefers the extra space, however.